doc/user/content/manage/access-control/service-accounts.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Service accounts"
+description: "Configure a new service account (i.e., non-human user) to connect external applications and services to Materialize."
+---
+
+It's a best practice to use service accounts (i.e., non-human users) to connect
+external applications and services to Materialize. As an **administrator** of a
+Materialize organization, you can create service accounts manually via the
+[Materialize console](https://console.materialize.com/), or programatically
+via [Terraform](/manage/terraform/).
+
+More granular permissions for the service account can then be configured using
+[role-based access control (RBAC)](/manage/access-control/#role-based-access-control-rbac).
+
+## How to create a service account
+
+### Materialize Console
+
+1. [Log in to the Materialize Console](https://console.materialize.com/).
+
+1. In the side navigation bar, click **+ Create New** > **App Password**.
+
+1. In the **New app password** modal, select **Type** > **Service** and name the
+new app password. Under **User**, specify the new service user you'd like to
+create and associate with the new app password.
+
+1. Optionally, associate the new service user with existing **Roles** to grant
+it existing database privileges. You can manage these privileges at any time
+using [RBAC](/manage/access-control/#role-based-access-control-rbac).
```

Comment:
The roles in the service account dialog aren't the database roles that are described in the RBAC docs. They're just the organization roles: "Organization Member" and "Organization Admin:

![image](https://github.com/user-attachments/assets/c2da1e32-bfc1-4cc0-94dc-2d413e725d34)

To grant database roles to the service user via RBAC, you need to log in to the SQL interface and run `GRANT` commands.

Reply:
Err, shouldn't have just trusted my memory here and paid attention to the Terraform code — thank you. 🌫️